### PR TITLE
chore: change refs of master to main

### PR DESCRIPTION
**Pull request by sherwinski**
This PR changes any references from the `master` branch to the `main` branch, now that it is the default.

### CODE REVIEW

**Review by ericdeansanchez — APPROVED** [review]
👍 LGTM!

File: setup.py [hunk 1]
```diff
@@ -16,9 +16,11 @@
     license='ISC',
     description='An app for integrating imgix into Django sites',
     long_description=README,
-    url='https://github.com/pancentric/django-imgix',
+    url='https://github.com/imgix/django-imgix',
     author='Pancentric Ltd',
     author_email='devops@pancentric.com',
```
Comment (ericdeansanchez) [inline setup.py:21]: More of a question... do we plan on changing these `author` fields or?

**Comment by frederickfogerty** [comment]
👏

File: setup.py (hunk 1 above)
Comment (sherwinski) [reply setup.py:21]: IMO I don't think we should. I think it's important to give credit to the original authors even if they will not be involved with the future of the library. For our purposes, I think the `maintainer` and `maintainer_email` fields are completely fine. [[1](https://docs.python.org/2/distutils/setupscript.html#additional-meta-data)]

> Either the author or the maintainer must be identified. If maintainer is provided, distutils lists it as the author in PKG-INFO.

**Pull request merged**